### PR TITLE
Improve the quiz12 with tests

### PR DESCRIPTION
**Pull request by y-martinez**
## Summary

This PR Close: y-martinez/PyZombis#16

## Checklist

- [X] Variables, functions and comments are translated to Spanish
- [X] Functions follow underscore notation
- [X] Spell check done & typos fixed
- [X] All python code is PEP8 compliant
- [X] Test coverage with Playwright implemented; locators are Pyhton code
- [X] Reviewers assigned (all peers & at least 1 mentor)

### Test Quiz12 exercise 1
test_quiz12_1_chromium

### CODE REVIEW

**Review by angelasofiaremolinagutierrez — COMMENTED** [review]
I left a suggestion ;)

File: _sources/quiz/Quiz12.rst [hunk 1]
```diff
@@ -2,171 +2,250 @@
...
-.. activecode:: verbing
-   :nocodelens:
+..  activecode:: q12_1
+    :nocodelens:
 
-   G. verbing |br|
-   Dada uma string, caso seu comprimento seja pelo menos 3, |br|
-   adiciona 'ing' no final |br|
-   Caso a string já termine em 'ing', acrescentará 'ly'. |br|
+    Desarrolle la función ``verbo`` que recibe una cadena ``s`` como parámetro. Si la longitud de la cadena es al menos 3, debe devolver la cadena original concatenando ``"ing"`` al final. Si la cadena ``s`` ya termina en ``"ing"``, concatene la cadena ``"ly"``. Si la longitud de la cadena en menor 3 devuelve la cadena original. |br| |br|
```
Comment (LeoCumpli21) [inline _sources/quiz/Quiz12.rst:15]:
```suggestion
    Desarrolle la función ``verbo`` que recibe una cadena ``s`` como parámetro. Si la longitud de la cadena es al menos 3, debe devolver la cadena original concatenando ``"ing"`` al final. Si la cadena ``s`` ya termina en ``"ing"``, concatene la cadena ``"ly"``. Si la longitud de la cadena es menor que 3, devuelve la cadena original. |br| |br|
```

**Review by LeoCumpli21 — APPROVED** [review]
I left one suggestion but in general LGTM :)

File: _sources/quiz/Quiz12.rst [hunk 2]
```diff
@@ -2,171 +2,250 @@
...
+
+    class myTests(TestCaseGui):
+        def testOne(self):
+
+            self.assertEqual(inicio_potencia(7), 46, "Esperado: 46")
+            self.assertEqual(inicio_potencia(3), 5, "Esperado: 5")
+            self.assertEqual(inicio_potencia(133), 316, "Esperado: 316")
+            self.assertEqual(inicio_potencia(1024), 10, "Esperado: 10")
+            self.assertEqual(inicio_potencia(123), 90, "Esperado: 90")
+            self.assertEqual(inicio_potencia(1), 0, "Esperado: 0")
```
Comment (NicolasSandoval) [inline _sources/quiz/Quiz12.rst:245]: The expected answer is not '4'?

Comment (y-martinez) [reply _sources/quiz/Quiz12.rst:245]: Yes could be, but the first one is ``2^0 = 1``

Comment (NicolasSandoval) [reply _sources/quiz/Quiz12.rst:245]: you're right in that case, my proof started from exponent 1.

**Review by NicolasSandoval — APPROVED** [review]

**Pull request merged**